### PR TITLE
upgraded to kombu 5, cleared bandit warnings

### PR DESCRIPTION
**Pull request by mchenchen**
for @JoshPaulsen

### CODE REVIEW

File: tests/utils/fakeredis_ext.py [hunk 1]
```diff
@@ -109,6 +114,7 @@ def __init__(self, client):
             self.client = client
             self._sock = self._socket()
             self.pid = 1234
+            self.retry = Retry(NoBackoff(), 0)
```
Comment (mchenchen) [inline tests/utils/fakeredis_ext.py:117]: This change is the result of this change in the redis library: https://github.com/redis/redis-py/pull/1494/files#diff-3161b005adfa1f5266a9736bffb5ede9d14e2ab40937cb241bc16ca5647df79dR974. It now expects a retry attribute on all connections.

File: kombu_redis_priority/transport/redis_priority_async.py [hunk 2]
```diff
@@ -668,7 +665,7 @@ def get_table(self, exchange):
         with self.conn_or_acquire() as client:
             values = client.smembers(key)
             if not values:
-                raise InconsistencyError(NO_ROUTE_ERROR.format(exchange, key))
+                return []
```
Comment (mchenchen) [inline kombu_redis_priority/transport/redis_priority_async.py:668]: I'm guessing that this code was originally copied from kombu's code so I looked at the reason why `NO_ROUTE_ERROR` doesn't exist anymore and it looks like in v5 they think it makes more sense just to return an empty list: https://github.com/celery/kombu/commit/73b10a93464bcf91126c2cf35dbdc096073789df#diff-203e309d100714904255d0af4500dd295e40e4ffedfdc9295ab3b386718ca87eL1000.

File: tests/utils/fakeredis_ext.py [hunk 3]
```diff
@@ -40,9 +42,9 @@ def _parse_pipeline_response_from_connection(self, connection):
         - QUEUED : acknowledges a command has been queued. There will be one per command sent.
         - LIST : list of responses
         """
-        # pop off the first command, which should be MULTI to signal start of transaction
         cmd = self.connection._sock.data.pop(0)
-        assert cmd[0] == 'MULTI'
+        if cmd[0] != 'MULTI':
```
Comment (mchenchen) [inline tests/utils/fakeredis_ext.py:46]: These conversions from assert to conditional exceptions are meant to appease bandit.

**Review by JoshPaulsen — APPROVED** [review]
This seems to work with https://github.com/Captricity/Cipher/pull/2362 !

**Pull request merged**